HHS/Head-Start-TTADP
# [TTAHUB-1827] "View" training reports

## Description of change
- Add a way to view training and session reports in a read only mode at ```/training-report/view/${id}```
- Added the "View event" button to the training reports table
- Also added two hooks to the backend for sessionReports. One that automatically moves a "not started" training report to "in progress" a session is created for it. The other prevents changing or deleting a session if the training report is complete.

## How to test
- Fill out a training report and some session reports
- Visit the training report view link and make sure it looks correct and matches the mockup
- Tests out the hooks - add a session to a training report and confirm the status is updated. Destroy or update a session on a completed training report.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1827


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
